94-appendixD.Rmd

```diff
@@ -815,5 +813,192 @@ library(ggplot2)
 library(dplyr)
 library(moderndive)
 library(gapminder)
-#library(skimr)
+library(skimr)
 ``` 
+
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Conduct a new exploratory data analysis with the same outcome variable $y$ being `score` but with `age` as the new explanatory variable $x$. Remember, this involves three things:
+
+a) Looking at the raw data values.
+a) Computing summary statistics.
+a) Creating data visualizations.
+
+What can you say about the relationship between age and teaching scores based on this exploration?
+
+**Solution**: 
+
+- Looking at the raw data values: 
+```{r}
+glimpse(evals_ch5)
+```
+
+- Computing summary statistics:
+```{r}
+skim_with(numeric = list(hist = NULL), integer = list(hist = NULL))
+evals_ch5 %>%
+  select(score, age) %>%
+  skim()
+```
+(Note that for formatting purposes, the inline histogram that is usually printed with skim() has been removed. This can be done by running `skim_with(numeric = list(hist = NULL), integer = list(hist = NULL))` prior to using the `skim()` function as well.)
+
+- Creating data visualizations:
+```{r}
+ggplot(evals_ch5, aes(x = age, y = score)) +
+  geom_point() +
+  labs(x = "Age", y = "Teaching Score",
+       title = "Scatterplot of relationship of teaching score and age")
+```
+
+
+<!--
+TODO: Albert needs to double check interpretation:
+-->
+
+
+Basid on the scatterplot visualization, there seem to have a weak negative relationship between age and teaching score. As age increases, the teaching score see, to decrease slightly. 
+
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Fit a new simple linear regression using `lm(score ~ age, data = evals_ch5)` where `age` is the new explanatory variable $x$. Get information about the "best-fitting" line from the regression table by applying the `get_regression_table()` function. How do the regression results match up with the results from your earlier exploratory data analysis?
+
+**Solution**: 
+```{r}
+# Fit regression model:
+score_age_model <- lm(score ~ age, data = evals_ch5)
+# Get regression table:
+get_regression_table(score_age_model)
+```
+$$
+\begin{aligned}
+\widehat{y} &= b_0 + b_1 \cdot x\\
+\widehat{\text{score}} &= b_0 + b_{\text{age}} \cdot\text{age}\\
+&= 4.462 - 0.006\cdot\text{age}
+\end{aligned}
+$$
+
+<!--
+TODO: Albert will verify Starry's interepretation:
+-->
+
+For every increase of 1 unit in `age`, there is an *associated* decrease of, *on average*, 0.006 units of `score`. It matches with the results from our earlier exploratory data analysis. 
```

Comment:
Ah! This is a subtle one, but check out #152: the correct interpretation is "for teachers who differ by 1 year in age" and not "as teachers age by 1 year"